device/project.godot

```diff
@@ -10,7 +10,7 @@ config_version=3
 
 [application]
 
-config/name="Prototype"
+config/name="Prototype_Escoria"
```

Comment:
Could you change this to "Escoria Prototype" instead, without the underscore, since that reads better`